front/lib/api/assistant/conversation/destroy.ts

```diff
@@ -154,6 +155,9 @@ export async function destroyConversation(
     await UserMessage.destroy({
       where: { id: userMessageIds },
     });
+    await AgentMessageFeedback.destroy({
+      where: { agentMessageId: agentMessageIds },
+    });
```

Comment:
Don't we need to do something similar in temporal/activities deleteConversationsActivity ?